jest.config.js

```diff
@@ -28,5 +28,8 @@ module.exports = {
                 output: './reports/junit/jest-results.xml'
             }
         ]
-    ]
+    ],
+    fakeTimers: {
+        enableGlobally: true,
+    }
```

Comment:
Without this, if we run the tests from inside VSCode then we get a bunch of errors similar to below (this does not happen if we run tests from command line via `yarn test`). According some SO posts, the solution is to use Fake Timers for the affected tests and since in our case it's all of the tests, I'm applying this globally.

```
ReferenceError: You are trying to `import` a file after the Jest environment has been torn down. From src/cli/commands/force/lightning/lwc/__tests__/preview.test.ts.

      at Object.get stripAnsi [as stripAnsi] (node_modules/@oclif/core/lib/cli-ux/deps.js:6:16)
      at SpinnerAction._lines (node_modules/@oclif/core/lib/cli-ux/action/spinner.js:73:14)
      at SpinnerAction._reset (node_modules/@oclif/core/lib/cli-ux/action/spinner.js:67:28)
      at SpinnerAction._render (node_modules/@oclif/core/lib/cli-ux/action/spinner.js:57:14)
      at Timeout._onTimeout (node_modules/@oclif/core/lib/cli-ux/action/spinner.js:28:67)
Waiting for the debugger to disconnect...
/Users/maliroteh/Depot/MPE/sfdx/lwc-dev-mobile/node_modules/@oclif/core/lib/cli-ux/action/spinner.js:73
            .stripAnsi(s)
```

Reply:
Verified that `stripAnsi` bug doesn't happen anymore on my machine.
![image](https://user-images.githubusercontent.com/42476999/214392043-462df520-dc2e-4e67-bb87-3aa452e5dbb4.png)